### PR TITLE
Only store policy data once

### PR DESCRIPTION
**Pull request by joejstuart**
Only collect the policy data once for each source group. Before this the data was duplicated for each component.

https://issues.redhat.com/browse/EC-1027

### CODE REVIEW

File: cmd/validate/image.go [hunk 1]
```diff
@@ -400,7 +400,7 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 			close(jobs)
 
 			var components []applicationsnapshot.Component
-			var manyData [][]evaluator.Data
+			var evaluatorData [][]evaluator.Data
```
Comment (lcarva) [inline cmd/validate/image.go:403]: Should this be `[]evaludator.Data` instead?

Comment (joejstuart) [reply cmd/validate/image.go:403]: Intuitively, I think so, but I don't know if it adds any value and there's an acceptance test for it even, so I'd prefer to leave it alone.

**Review by lcarva — APPROVED** [review]

**Review by zregvart — APPROVED** [review]
One followup could be that we only collect the data if `--output=data` or the like was provided

**Comment by joejstuart** [comment]
> One followup could be that we only collect the data if `--output=data` or the like was provided

I think this will do that. https://github.com/enterprise-contract/ec-cli/pull/2184/commits/5464dbfd7a3cf8808e64e49c23badd390b2ff2bb

**Pull request merged**